micronota/parsers/gff.py

```diff
@@ -0,0 +1,297 @@
+r'''
+GFF3 Parser
+=====================
+
+GFF is a standard file format for storing genomic features in a text file.
+GFF stands for Generic Feature Format. GFF files are plain text, 9 column,
+tab-delimited files [#]_.
+
+The first line of the file is a comment that identifies the format and version.
+This is followed by a series of data lines, each one of which corresponds
+to an annotation.
+The 9 columns of the annotation section are as follows:
+
+    +--------------------------------+----------------------------------+
+    | SEQID - ID of the landmark used| (begins each entry; 1 per entry) |
+    +--------------------------------+----------------------------------+
+    | SOURCE - algorithms used       | 1 per entry                      |
+    +--------------------------------+----------------------------------+
+    | TYPE - type of the feature     | 1 per entry                      |
+    +--------------------------------+----------------------------------+
+    | START - start of the feature   | 1 per entry                      |
+    +--------------------------------+----------------------------------+
+    | END - end of the feature       | 1 per entry                      |
+    +--------------------------------+----------------------------------+
+    | SCORE - floating point score   | 1 per entry                      |
+    +--------------------------------+----------------------------------+
+    | STRAND - +/-/./?               | 1 per entry                      |
+    +--------------------------------+----------------------------------+
+    | PHASE - only for TYPE="CDS"    | 1 per entry                      |
+    +--------------------------------+----------------------------------+
+    | ATTR - feature attributes (tag)| 1 per entry                      |
+    +--------------------------------+----------------------------------+
+
+
+Column 9 (ATTR) tags have predefined meanings:
+
+ID
```

Comment:
The following line will be concatenated to this header into one line. Consider other formatting options (http://docutils.sourceforge.net/docs/user/rst/quickref.html)?